OOPConcepts/src/com/temperature/Temperature.java

```diff
@@ -0,0 +1,44 @@
+package com.temperature;
+
+public class Temperature {
+
+	private double number;
+
+	private TempScale scale;
+
+	public Temperature() {
+		number = 0.0;
+		scale = TempScale.FAHRENHEIT;
+	}
+
+	public Temperature(double number) {
+		this.number = number;
+		scale = TempScale.FAHRENHEIT;
+	}
+
+	public Temperature(TempScale scale) {
+		number = 0.0;
+		this.scale = scale;
+	}
+
+	public Temperature(double number, TempScale scale) {

```

Comment:
this constructor should do the work of other constructors.

Reply:
Absolutely. But I was using various ways of using/creating different constructors.

Reply:
ok I understand the purpose now.